lib/utils/getAPI.dart

```diff
@@ -1,29 +1,107 @@
-/*import 'package:http/http.dart' as http;
+import 'dart:io';
```

Comment:
The name of this file should be updated since you have multiple types of requests. You can keep all the requests in one file, but personally I think it would make sense to break up the requests into category of request per file:
- userAPI
- foodAPI
- ...

One per main section of the [backend API](https://app.swaggerhub.com/apis/mplekunov/Smart-Chef/1.0.0#/)

And then you can have an APIutil.dart file that has the base request types, the `API_PREFIX` var, everything you need to use in the downstream API files.